Bumerang/app/src/main/java/com/seng480b/bumerang/Offer.java

```diff
@@ -0,0 +1,61 @@
+package com.seng480b.bumerang;
+
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+public class Offer {
+    private int offer_id;
+    private Profile offer_profile;
+    private Request offerReq;
+
+    public Offer(Profile offer_profile,
+                 Request request, int id) {
+        this.offer_profile = offer_profile;
+        this.offerReq = request;
```

Comment:
In Java, camel case should be used for variables and data members. So instead of offer_profile it should be offerProfile, same with offerId.